File: algorithms/interview-preparation-kit-1/ruby/code-response/spec/ZeroMeansZero/zero_means_zero.spec.rb
require_relative "../../src/ZeroMeansZero/zero_means_zero"

describe ZeroMeansZero do
  describe ".noZero" do
    context "Sum without zero" do
      it "return 5" do
        expect(ZeroMeansZero.noZero(5,45)).to eql(5)
      end
    end
  end
end

File: algorithms/interview-preparation-kit-1/php/code-response/spec/ZeroMeansZero/ZeroMeansZero.spec.php
<?php

use \ZeroMeansZero\ZeroMeansZero;

describe('Zero means Zero', function () {

  it('remove zero of 7 + 8', function () {
    expect(ZeroMeansZero::change(7, 8))->toBe('15');
  });

  it('remove zero of 15 + 5', function () {
    expect(ZeroMeansZero::change(15, 5))->toBe('2');
  });

  it('remove zero of 99 + 6', function () {
    expect(ZeroMeansZero::change(99, 6))->toBe('15');
  });

});
